cf-java-logging-support-opentelemetry-agent-extension/README.md

```diff
@@ -0,0 +1,121 @@
+# OpenTelemetry Java Agent Extension for SAP Cloud Logging
+
+This module provides an extension for the [OpenTelemetry Java Agent](https://opentelemetry.io/docs/instrumentation/java/automatic/).
+The extension scans the service bindings of an application for SAP Cloud Logging.
```

Comment:
Link to SAP Cloud Logging?
(I know the doc as well as sap cloud logging is not GA yet. So is this a precondition for this PR?)

Reply:
synced w/ @KarstenSchnitter: link will be provided soon once available